src/models/event.ts

```diff
@@ -394,5 +413,17 @@ export default class Event extends RestfulModel {
         'The number of participants in the event exceeds the set capacity.'
       );
     }
+
+    if (
+      this.id &&
+      ((this.reminders &&
+        (this.reminders.reminderMethod || this.reminders.reminderMinutes)) ||
+        this.reminderMethod ||
+        this.reminderMinutes)
+    ) {
```

Comment:
After we make the reminders object readonly we can omit it from this check.

```suggestion
    if (
      this.id &&
        this.reminderMethod ||
        this.reminderMinutes)
    ) {
```